### DISCUSSION TITLE
How can I open different project addresses in the same TAURI app and both need to call window.__TAURI__

### DISCUSSION THREAD

**Discussion by Soya-xy** [tauri-apps/tauri#4480] [Q&A]
When I use tauri::Windows::WindowBuilder create a new window to the URL is not the current devPath (eg: http://localhost:3000/), but another project (eg: http://localhost:3333/), the other items window.__TAURI__ is undefined if you want to use the tauri API.

Is this currently supported?

All projects run in the same Tauri app, but there are 2 different project addresses.

**Comment by FabianLars** [comment]
The next time you have a question, please don't ask it in 3 places at the same time. For completeness sake, [here is the answer](https://github.com/tauri-apps/tauri/issues/4479#issuecomment-1166790492) to your question:
> You'll need to merge both somehow. For instance you can use Webpack dev proxy in development mode, and move all assets to the same folder for production. We don't support multiple dev URLs currently.
